src/client/app/components/MultiCompareChartComponent.tsx

```diff
@@ -34,14 +35,20 @@ export default function MultiCompareChartComponent(props: MultiCompareChartProps
 
 	return (
 		<div className='row'>
-			{props.selectedMeters.map(meterID =>
-				<div className={childClassName} key={meterID}>
-					<CompareChartContainer key={meterID} id={meterID} isGroup={false} />
+			{props.selectedMeters.map(meter =>
+				<div className={childClassName} key={meter.id}>
+					<CompareChartContainer
+						key={meter.id}
+						entity={meter}
+					/>
 				</div>
 			)}
-			{props.selectedGroups.map(groupID =>
-				<div className={childClassName} key={groupID}>
-					<CompareChartContainer key={groupID} id={groupID} isGroup />
+			{props.selectedGroups.map(group =>
```

Comment:
You are probably going to want to change some logic here, because this always puts meters (sorted in a custom order) first, then groups (sorted in the custom order). You probably want to merge the two together.

Reply:
I concur with this. I think we should treat meters and groups the same way. There are likely changes coming that will reinforce this view of wanting to graph them as equals.